airqo-platform/AirQo-frontend
# [Website] Update Profile endpoint

#### Summary of Changes (What does this PR do?)
- Updates profile endpoint to include new profile field

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch and checkout to this branch locally
* Follow instructions in the `README.md` file to setup and run the app.
* Confirm the summary of changes listed above
* Check out the the api docs and try run the team GET endpoint. The swagger docs url - http://localhost:8000/api/
* A sample profile can be created from the admin portal. Portal url - http://localhost:8000/admin/
* To create a super user for portal login, run `python manage.py createsuperuser` in the app console and follow the prompt.

#### What are the relevant tickets?
- [WEB-220](https://airqoteam.atlassian.net/browse/WEB-220)
